src/Snippets/Docs/Migration.Retry.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Polly.Retry;
+using Snippets.Docs.Utils;
+
+namespace Snippets.Docs;
+
+internal static partial class Migration
+{
+    public static void Retry_V7()
```

Comment:
Could be kind of overwhelming with so many options, maybe could be broken down into individual snippets? Like if you use this it migrates to this. More work but probably easier to understand.

Reply:
True, split the samples into more sections